cave/utils/converter/hpbandster2smac.py

```diff
@@ -0,0 +1,99 @@
+import os
+import logging
+import tempfile
+
+from ConfigSpace.read_and_write import pcs_new
+from smac.configspace import Configuration, ConfigurationSpace
+from smac.tae.execute_ta_run import StatusType
+from smac.runhistory.runhistory import RunHistory
+from smac.optimizer.objective import average_cost
+from smac.scenario.scenario import Scenario
+from smac.stats.stats import Stats
+from smac.utils.io.output_writer import OutputWriter
+from smac.utils.io.traj_logging import TrajLogger
+
+class HpBandSter2SMAC(object):
+
+    def __init__(self):
+        self.logger = logging.getLogger(self.__module__ + '.' + self.__class__.__name__)
+
+    def convert(self, folder):
+        try:
+            from hpbandster.core.result import Result as HPBResult
+            from hpbandster.core.result import logged_results_to_HB_result
+        except ImportError as e:
+            raise ImportError("To analyze BOHB-data, please install hpbandster (e.g. `pip install hpbandster`)")
```

Comment:
This will quickly be useless code if hpbandster is a requirement.
Should hpbandster not be a requirement?

Reply:
I don't know. Right now I'm not sure how stable the installation is. When installing via pip, I need to install lots of dependencies afterwards. I wouldn't want to break CAVE's workflow. Also adding a big dependency for just one usecase seems overkill. But then again, I don't know what's best-use-practice here. Might as well add as requirement.

Reply:
Then for now this should do. But you should see that the Import Error is thrown as early as possible.